File: cs/Backend/Domain/Models/AnalyzedPost.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;

namespace Socneto.Domain.Models
{
    public class AnalyzedPost
    {
        public Guid JobId { get; set; }

        [JsonProperty("postDto")]
        public Post Post { get; set; }

        public Dictionary<string,  Dictionary<string, AnalysisValue>>[] Analyses { get; set; }
    }
}

File: frontend/lib/src/models/AnalyzedPost.dart
import 'package:sw_project/src/models/Post.dart';

class AnalyzedPost {
  final String jobId;
  final Post post;
  final dynamic analyses;

  AnalyzedPost(this.jobId, this.post, this.analyses);

  AnalyzedPost.fromMap(Map data) :
        jobId = data["job_id"],
        post = Post.fromMap(data["post"]),
        analyses = data["analyses"];
}